CMakeFiles/3.16.0-rc3/CMakeCXXCompiler.cmake

```diff
@@ -0,0 +1,88 @@
+set(CMAKE_CXX_COMPILER "C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterprise/VC/Tools/MSVC/14.16.27023/bin/Hostx64/x64/cl.exe")
```

Comment:
Why so many CMake changes?

Reply:
That Cmake folder was clearly not supposed to be commited... 😞